person_capture/gui_app.py

```
@@ -9913,6 +9945,84 @@ def _validate_hdr_sdr_export_image(
+
+            bad = (
+                (rect_count >= 80 and rect_area >= 5000)
+                or (rect_count >= 50 and drop_count >= 12000)
+            )
+            if not bad:
+                return False, ""
+            return True, (
+                f"dark_block_dropouts pixels={drop_count} "
+                f"components={rect_count} area={rect_area}"
+            )
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)